pkg/ebpf/tracer_windows.go

```diff
@@ -45,7 +172,46 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 
 // GetStats returns a map of statistics about the current tracer's internal state
 func (t *Tracer) GetStats() (map[string]interface{}, error) {
-	return nil, ErrNotImplemented
+	if t.driverHandle == windows.InvalidHandle {
+		return nil, fmt.Errorf("Problem with handle cannot get stats.")
+	}
+
+	var (
+		bytesReturned uint32
+		stats         C.struct_driver_stats
+		statbuf       = make([]byte, C.sizeof_struct_driver_stats)
+		ioctlcd       = ctl_code(NETWORK_DEVICE_TYPE_CTL_CODE, DDFILTER_IOCTL_GETSTATS, uint32(0), uint32(0))
+	)
+
+	err := windows.DeviceIoControl(t.driverHandle, ioctlcd, nil, 0, &statbuf[0], uint32(len(statbuf)), &bytesReturned, nil)
+	if err != nil {
+		log.Errorf("Error reading Stats with DeviceIoControl: %v", err)
```

Comment:
I think you'll want to return the error here, otherwise you'll get unexpected results.